### ISSUE TITLE
Move this repo to countries org?

### ISSUE THREAD

**Issue by scudco** [stefanpenner/country_select#193]
@stefanpenner @pmor since there is now a https://github.com/countries would it make sense to move this repo over there? I don't have access to administer this repo, but it seems it would make more sense to put it under an org with more active management. It'd be nice to have it live alongside its dependent gem.

@pmor I can't see any details of the `countries` org but maybe you can tag the correct person in here if you're not them.

**Comment by pmor** [comment]
@scudco Yep, I'm one of the admins of the `countries` org, but I'll tag @rposborne and @hexorx all the same in case they want to add to this.

On my part, I'd be happy to have this gem come under the org and help maintain it.

**Comment by stefanpenner** [comment]
Sure! Should I transfer it?

**Comment by pmor** [comment]
@stefanpenner Feel free to transfer it. I'm adding you and @scudco to the org. Are there other maintainers that I should add?

**Comment by stefanpenner** [comment]
When I attempt to transfer I get: `You don’t have the permission to create public repositories on countries` Could you temporarily give me permission on the countries org to create public repos?

**Comment by pmor** [comment]
@stefanpenner I invited you as an org owner, but I don't think you've accepted the invitation yet

**Comment by stefanpenner** [comment]
Transfer complete
